src/main/scala/com/taxis99/amazon/sqs/SqsConfig.scala

```diff
@@ -14,7 +14,7 @@ protected[this] trait SqsConfig {
     * Defines the serialization method to produce/consume messages.
     * @return The serialization object
     */
-  def serializer: ISerializer = PlayJson
+  implicit def serializer: ISerializer = PlayJson
```

Comment:
It actually don't, IDK if it's for the better or worse but Scala (both 2.11 and 2.12) compiler actually inherited these parameter from the base blas even so when it's not explicit wirtten. The actual integration tests shows that behaviour.